docs/mobile-apps/automated-testing/appium.md

```diff
@@ -1,437 +1,141 @@
 
-**Android**
-```py
-desired_caps = {}
-desired_caps['platformName'] = 'Android'
-desired_caps['app'] = PATH('../../apps/MyIOS.apk')
-self.driver = webdriver.Remote('https://ondemand.us-west-1.saucelabs.com/wd/hub', desired_caps)
 ```
-
-</TabItem>
-<TabItem value="Node.js">
-
-We recommend using the [webdriverio testing utility](https://webdriver.io/) as your WebDriver testing framework for node.js. For more information, refer to the [documentation](https://webdriver.io/docs/gettingstarted/).
-
-**iOS**
-```js
-const opts = {
-  capabilities: {
-    platformName: "iOS",
-    app: "/path/to/the/downloaded/MyiOS.app.zip",
-  }
-};
-
-const client = wdio.remote(opts);
+deviceName: “Android GoogleAPI Emulator”
+platforName: “Android”
+automationName: “UiAutomator2”
 ```
 
-**Android**
-```js
-const opts = {
-  capabilities: {
-    platformName: "Android",
-    app: "/path/to/the/downloaded/MyAndroid.apk",
-  }
-};
+### Step 4: Add Capabilities for the Browser
+Depending on which browser you would like to test on, you need to [set the capabilities](https://docs.saucelabs.com/dev/test-configuration-options#desktop-browser-capabilities-sauce-specific--optional) to test on that browser:
 
-const client = wdio.remote(opts);
 ```
-
-</TabItem>
-<TabItem value="Ruby">
-
-Visit [appium_lib](https://github.com/appium/ruby_lib/blob/master/lib/appium_lib/driver.rb) for more information about the Ruby Appium language bindings.
-
-**iOS**
-```ruby
-opts = { caps: { platformName: :ios, app: '/path/to/MyiOS.app'}, appium_lib: { wait_timeout: 30 }}
-appium_driver = Appium::Driver.new(opts, true)
-appium_driver.start_driver
-```
-
-**Android**
-```ruby
-opts = { caps: { platformName: :android, app: '/path/to/MyAndroid.apk'}, appium_lib: { wait_timeout: 30 }}
-appium_driver = Appium::Driver.new(opts, true)
-appium_driver.start_driver
-```
-
-</TabItem>
-<TabItem value="C#">
-
-Visit the [appium-dot-net page](https://github.com/appium/appium-dotnet-driver) for more information about the C# Appium language bindings.
-
-**iOS**
-```CS
-var sauceURL = "https://ondemand.us-west-1.saucelabs.com";
-var driver = new IOSDriver<IWebElement>(new Uri(sauceURL));
-```
-
-**Android**
-```CS
-var sauceURL = "https://ondemand.us-west-1.saucelabs.com";
-var driver = new AndroidDriver<IWebElement>(new Uri(sauceURL));
+browserName: "Chrome"
 ```
+At a minimum, you must set the `browserName` capability, however different browsers have other options you can set.
 
-</TabItem>
-</Tabs>
-
-<br/>
-
-### 3. Locate App Elements
+<img src={useBaseUrl('img/mobile-apps/appium-web-capabilities.png')} alt="Appium Capabilities" width="800"/>
 
-Next, locate an element within your app, such as a login button or a link.
-In order to find elements in a mobile environment, Appium implements a number of locator strategies that are specific to, or adaptations for, the particulars of a mobile device. Three are available for both Android and iOS:
+## Test a Mobile App on Sauce Labs Real Devices
```

Comment:
Why are we talking about `Sauce Labs Real Devices` if the intro only mentions EMUSIM?

Reply:
Updated the intro to this:

This page is intended to provide you with instructions on how to quickly set up Appium tests with Java using the TestNG test runner on an Android Emulator or an iOS Real Deives, and run tests using the Sauce Labs App with the Virtual Device and Real Device Cloud. To see a comprehensive list of test examples, [check out the appium directory in our demo-java repo](https://github.com/saucelabs-training/demo-java/tree/dff5fd61b8e152efe59e4a8c9e75c644de4e51e0/appium-examples).